app/src/components/items/viewer/viewer.tsx

```diff
@@ -0,0 +1,71 @@
+"use client";
+
+import AudioViewer from "../clover/audio/viewer";
+import VideoViewer from "../clover/video/viewer";
+import { ItemModel } from "../../../models/item";
+import React from "react";
+import { UniversalViewer } from "../uv/universalViewerLazy";
+import "universalviewer/dist/esm/index.css";
+
+interface ItemProps {
+  item: ItemModel;
+  type: string;
+}
+
+const contentTypes = {
+  "still image": "image",
+  "moving image": "video",
+  "sound recording": "audio",
+};
+
+const ItemMediaViewer = ({ item, type }: ItemProps) => {
+  console.log("Item metadata: ", item.metadata);
+  let viewer;
+  switch (type) {
+    case "image":
+      if (item.isSingleCapture) {
+        viewer = (
+          <>
+            <UniversalViewer
+              manifestId={item.manifestURL}
+              // "http://localhost:8000/items/8e8dc6a0-c6eb-012f-7a58-58d385a7bc34"
+              // "https://wellcomelibrary.org/iiif/b18035723/manifest"
+              // manifestId={`http://localhost:8000/items/${item.uuid}`} //{"https://wellcomelibrary.org/iiif/b18035723/manifest"}
+              canvasIndex={0}
+              config={{}}
+            />
+          </>
+        );
+      } else {
+        viewer = (
+          <>
+            <UniversalViewer
+              manifestId={item.manifestURL}
+              // {`${process.env.COLLECTIONS_API_URL}/manifests/${item.uuid}`} //{"https://wellcomelibrary.org/iiif/b18035723/manifest"} //{`https://be73-100-37-199-113.ngrok-free.app/items/${item.uuid}`} //{"https://wellcomelibrary.org/iiif/b18035723/manifest"}
+              canvasIndex={0}
+              config={{}}
+            />
+          </>
+        );
+      }
```

Comment:
This isn't wrong, but consider another approach, in the if/else define the object data for the `UniversalViewer` component. Something like:
```
if (...) {
  dataObj = { ... }
} else {
  dataObj = { ... }
}
```

This is mostly to declare the component once and pass in the right props rather declaring it twice with different props.

```
case "image":
  if (...) {
    dataObj = { ... }
  } else {
    dataObj = { ... }
  }
  return <UniversalViewer {...dataObj} />;
case "video":
  ...
```